.github/workflows/python-package.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['2.7', '3.5', '3.6', '3.7', '3.8', '3.9']
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install -r requirements-dev.txt
+    - name: tox
+      run: |
+        tox -e `python -c "import sys; print('py' + ''.join(sys.version.split('.')[:2]))"`
```

Comment:
I think  `tox -e py` does the same thing:

> The environment py uses the version of Python used to invoke tox.

see https://tox.readthedocs.io/en/latest/example/basic.html#a-simple-tox-ini-default-environments

Reply:
Aha that's much nicer, thanks @kmike 👍